modules/round/src/main/GameProxyRepo.scala

```diff
@@ -58,4 +61,29 @@ final class GameProxyRepo(
         }
       }
     }
+
+  def correspondenceGamesToPlayOf(
```

Comment:
Should the games be fetched in the db directly? I though using game proxy would give better performance, but not sure.

Also we fetch games of all players at the same of day, which will produce a spike of activity. Another option would be to smooth that over the day, but I don't know how it'd be done